examples/go/word_count/count_this.txt

```diff
@@ -0,0 +1,29 @@
+Amendment I
```

Comment:
We can probably get rid of this whole file. I'm pretty sure I committed it by accident.

Reply:
Will do